src/cli/__tests__/cli.test.ts

```diff
@@ -47,3 +51,40 @@ describe('collect', () => {
     });
   });
 });
+
+describe('visualize', () => {
+  beforeEach(() => {
+    loadProjectStructure('typeScriptVisualizeProject');
```

Comment:
I wonder if we need to maintain a complete project like this to generate a couple of entities and relationships for testing the vis.js HTML generation. If the visualizer just took in a `FileSystemGraphObjectStore`, and that had the ability to iterate it's entities and relationships, then it would be pretty easy to pass in an in-memory store and see the generated HTML in a test.

Reply:
In this test, I'm following the pattern of the `collect` command and creating an integration test where we see if the CLI works. I've already tested the lower level functions with mocked file systems to see if they work. If we want we could also just remove this test since there is one testing the integration between collect and visualize already. Not sure what you do think @aiwilliams.

Reply:
I think it's fine to keep, we'll see if it ever feels expensive to maintain! Thanks @softwarewright for considering.

Reply:
Considering @charlieduong94 comments about having the visualization tool work off any valid set of files, I'm growing convinced we can avoid running the `collect` command against a fixture integration, and instead just provide a memfs file setup to the generator.

Reply:
That makes sense, would we still want to have at least one integration test between collect and visualize to make sure we haven't broken the visualization with our own commands? Then we can figure out how we want to move this visualize test to memfs.

Reply:
I see what you mean, yeah, it would be good to know that our collection output can be visualized.